examples/systemd/user/buildkit.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=BuildKit
+Documentation=https://github.com/moby/buildkit
+
+[Service]
+Type=notify
+NotifyAccess=all
+ExecStart=rootlesskit /usr/local/bin/buildkitd --addr unix://%t/buildkit/rootless
```

Comment:
We should unshare the network namespace to protect the abstract sockets and localhost ports from BuildKit containers.

Basically, just `--net=slirp4netns --disable-host-loopback --copy-up=/etc` would be fine (https://github.com/moby/buildkit/blob/master/docs/rootless.md#running-buildkit-in-rootless-mode), but maybe we should have `buildkit-rootless.sh` to fix up `/etc/ssl` for openSUSE hosts

https://github.com/containerd/nerdctl/blob/4d486a5b0238ad1ac6557c434fdf2babbb6db67f/extras/rootless/containerd-rootless.sh#L135

Reply:
Ok, I think there was some systemd way to define such "defaults" in a nice looking way, will look it up

Reply:
There was no nice way, the systemd author says it should go into configuration files instead.

Reply:
http://0pointer.net/blog/projects/on-etc-sysinit.html